Testscripts/Windows/KDUMP-TestScript.ps1

```diff
@@ -116,7 +116,7 @@ function Main {
     }
     # Rebooting the VM in order to apply the kdump settings
     Run-LinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort `
-        -command "reboot" -runAsSudo -RunInBackGround | Out-Null
+        -command "sleep 5; reboot" -runAsSudo -RunInBackGround | Out-Null
```

Comment:
I think "sync" would be better than "sleep 5".

Reply:
@santoshx Thanks for your review. This is to fix that PS terminal lost Linux connections when execute reboot, for sync, it will write the cached data into disk, does it has the same effect with sleep?

Reply:
@LiliDeng No sync is no good in that case.